beheer.md

```diff
@@ -121,26 +121,36 @@ Beheren publicatie op GEMMA Online | Ja | Zelf
 Bij de ontwikkeling van Open API standaarden nemen diverse partijen deel. Het is handig voor ontwikkelaars maar ook voor beheerders om daar een overzicht van te hebben. Daarom dient de project manager of scrum master bij de aanvang van een ontwikkelproject maar ook tijdens het project als de teamsamenstelling wijzigt de [lijst met Deelnemende partijen](https://github.com/VNG-Realisatie/api-beheer/blob/master/Deelnemende%20partijen.md) in te vullen. Ook de beheerder of beheerders van de standaard moeten na in beheername van een standaard of bij wijziging van de beheerder op deze lijst worden vermeldt.
 
 ## Beheerproces
-Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
-Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam. Teneinde de API standaard breed gedragen en ondersteund te laten worden is een community van belang. Deze community wordt gevormd door gemeenten, leveranciers en beheer. 
+Vanuit en samen met die community wordt dat beheerteam daarna voor die betreffende standaard verantwoordelijk voor:
 1. beantwoorden van vragen;
 2. het oplossen van problemen;
 3. doorontwikkeling van de standaard.
 
 **Beantwoorden van vragen**
+Vragen mbt. de betreffende standaard kunnen beheer op verschillende manieren bereiken:
+* Mail/telefoon
+* Discussieforum _(NB doen we nog iets met een discussieforum of verlaten we dat pad?)_
```

Comment:
Volgens mij hadden we al besloten om voor de API standaarden niets te doen met de discussiefora maar alleen gebruik te maken van Slack en de discussiemogelijkheden in GitHub/GitLab.